park_api/cities/Heidelberg.py

```diff
@@ -0,0 +1,68 @@
+from bs4 import BeautifulSoup
+from park_api.util import convert_date
+from park_api.geodata import GeoData
+import datetime
+import json
+import urllib
+import random
+
+# This loads the geodata for this city if <city>.geojson exists in the same directory as this file.
+# No need to remove this if there's no geodata (yet), everything will still work.
+geodata = GeoData(__file__)
+
+# This function is called by the scraper and given the data of the page specified as source in geojson above.
+# It's supposed to return a dictionary containing everything the current spec expects. Tests will fail if it doesn't ;)
+def parse_html(html):
+
+    # BeautifulSoup is a great and easy way to parse the html and find the bits and pieces we're looking for.
+    # soup = BeautifulSoup(html, "html.parser")
+
+    # read the JSON-file:
+    #      URL + no caching
+    urlHD = "http://parken.heidelberg.de/api-v1/parking-location?api_key=H5WaIyR4lgn6wzo7rJf8u4ubecgpX0Q8&nc="+str(random.random())
```

Comment:
Ist der random String hier wirklich notwendig? Der Request bekommt ohne diesen Parameter doch auch eine Antwort.

```suggestion
    urlHD = "http://parken.heidelberg.de/api-v1/parking-location?api_key=H5WaIyR4lgn6wzo7rJf8u4ubecgpX0Q8
```

Reply:
im Original ist das auch drin.

Hier bei meinen Tests hatte ich das Problem, daß ich "uralte" Daten erhalten habe, d.h. >1 Stunde alt. Erst der Random-Teil löste das.

Reply:
Hui, dann ist das wohl doch relevant?

Reply:
Hab's jetzt ein paar mal lokal probiert, die Daten scheinen ohne den `nc` Parameter immer von vor maximal wenigen Minuten. Der Timestamp ist auch in der aktuellen Zeitzone hier. Eigenartig.

Reply:
OK, kann das auch nicht reproduzieren. Was auch immer ich da gesehen habe....

Ich nehm es raus.

Reply:
Aus meiner Sicht setzt der Server die Cache-Control header richtig mit max-age=60; must-revalidate.
Allerdings scheint trotz max-age=60 und einminütiger expires-Angabe serverseitig nur alle 3 Minuten aktualisiert zu werden. 

Ich gehe davon aus, das nc nicht mehr erforderlich. Vielleicht noch ein Relikt früherer Tage, als der Server keine entsprechenden Header übermittelt hat.